### DISCUSSION TITLE
Can I write app for Windows Phone 10 using MAUI

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#8485] [General]
Idk if I can publish MAUI / UWP to windows store for windows phone nowadays, Lumia still feels good with how light it is and the camera quality and new iphones are boring and a waste of money anyways, let me write app for my phone, make the developers happy (Mission) and let them make the users happy (Impact) and let them make the company happy (Prosperity)

**Comment by GreenShadeZhang** [comment]
I think you're a bad idea, wp itself is an outdated system that doesn't update, if you want to run maui I suggest you run Lumia 930 through a community project to run Windows 11, and then run maui.

**Reply by Xyncgas** [reply]
I thought windows operating systems are available on arm architectures now
